plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -224,6 +223,21 @@ public BigQueryConfig setCaseInsensitiveNameMatching(boolean caseInsensitiveName
         return this;
     }
 
+    @NotNull
+    @MinDuration("0ms")
+    public Duration getCaseInsensitiveNameMatchingCacheTtl()
```

Comment:
Should the user be allowed to set this value without `caseInsensitiveNameMatching` being set to `true` ?

```
io.trino.plugin.bigquery.BigQueryConfig#validate
```